packages/salesforcedx-vscode-core/src/util/orgUtil.ts

```diff
@@ -40,58 +41,61 @@ export async function checkForExpiredOrgs(orgList: OrgList) {
       daysUntilExpiration.getDate() + daysBeforeExpire
     );
 
-    const authInfoObjects = await orgList.getAuthInfoObjects();
-    if (!authInfoObjects) {
+    const orgAuthorizations = await AuthInfo.listAllAuthorizations();
+    if (!orgAuthorizations) {
       return;
     }
 
-    const orgsAboutToExpire = authInfoObjects!.filter((authInfoObject: FileInfo) => {
+    let results: string[] = [];
+    for (let i=0; i<orgAuthorizations.length; i++) {
```

Comment:
be sure to run the formatter...there's some space issues with this line.

Reply:
done